src/xhr.js

```diff
@@ -63,6 +79,46 @@ class Xhr {
 
 
 /**
+ * Normalized method name by uppercasing.
+ * @param {string|undefined} opt_method
+ * @return {string}
+ * @private
+ */
+export function normalizeMethod_(opt_method) {
```

Comment:
Just `method`. `opt_` indicates that it can be omitted as an argument call, but that's not the case - you always call it, even if the value `undefined`. It's the different between `string|undefined` and `string=`